src/shared/sam/localLambdaRunner.ts

```diff
@@ -195,8 +223,12 @@ export async function invokeLambdaFunction(
         debugPort: !config.noDebug ? config.debugPort?.toString() : undefined,
         debuggerPath: config.debuggerPath,
         extraArgs: config.sam?.localArguments,
+        parameterOverrides: config.parameterOverrides,
     }
 
+    // no longer necessary
+    delete config.invokeTarget
```

Comment:
the old comment was intentional and clearer:
```suggestion
     delete config.invokeTarget // Must not be used beyond this point.
```